### ISSUE TITLE
not equal enough

### ISSUE THREAD

**Issue by Toreole** [ErisBlastar/cplusequality#30]
"==" shuold be replaced with "==========" or any desired number of "="signs

**Comment by activedecay** [comment]
I fully agree that we should always strive to be __more__ equal. Now, how do you propose we handle the case where the opinion on either side of the equality is less equal? Or, Feminine forbid, if the opinion is patriarchal? How about a default short-circuit  to the side of whatever is deemed more equal?
